recipes/21cmFAST/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "21cmFAST" %}
+{% set version = "3.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6e88960d134e98e4719343d853c63fc3c691438b57b2863f7834f07fae9eab4f
+
+
+build:
+  number: 0
+  script: "INC=$(CONDA_PREFIX)/include LIB=$(CONDA_PREFIX)/lib {{ PYTHON }} -m pip install . -vv"
```

Comment:
Shouldn't these be `${PREFIX}`.  `CONDA_PREFIX` is something else and we don't want a subshell

Reply:
I'm not really sure on the difference between `PREFIX` and `CONDA_PREFIX`, but both seem to work OK, so I have switched to `PREFIX`.

Reply:
CONDA_PREFIX is where conda is installed. PREFIX is where the package should be installed to